addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -218,7 +217,7 @@ const FormEditor = options.Class.extend({
         if (!field.id) {
             field.id = weUtils.generateHTMLId();
         }
-        const params = { field: { ...field }, defaultName: escape(_t("Field")) };
```

Comment:
Note: I don't see where `defaultName` would be unsafely used, do I miss something?